mobile/android/logcat-helper.ts

```diff
@@ -48,6 +51,13 @@ export class LogcatHelper implements Mobile.ILogcatHelper {
 			this.mapDeviceToLoggingStarted[deviceIdentifier] = true;
 		}
 	}
+
+	public stop(deviceIdentifier: string): void {
+		let adbLogcat = this.adbLogCats[deviceIdentifier];
+		if (adbLogcat) {
+			adbLogcat.kill();
+		}
```

Comment:
You should consider removing `adbLogcat` from `this.adbLogCats` so that it can be garbage collected